.github/ISSUE_TEMPLATE/feature_request.yml

```
@@ -0,0 +1,58 @@
+---
+name: Airflow feature request
+description: Suggest an idea for this project or its docs
```

```suggestion
description: Suggest an idea for this project
```
the docs split out to `Airflow Doc issue report`